.github/workflows/unidoc.yaml

```diff
@@ -0,0 +1,19 @@
+  name: "Unidoc generation"
+  on: [pull_request]
```

Comment:
@vkorukanti Do we want it to also be on push? I think all our other jobs are

Reply:
yep, we should. Updating..